README.md

```diff
@@ -134,7 +134,12 @@ By default, `preact-cli` emulates the following config:
 ```
 
 #### Babel
-To customize babel simply create [```.babelrc```](https://babeljs.io/docs/usage/babelrc/) file. [Preact CLI preset] will be applied automatically so you won't have to worry about keeping your ```.babelrc``` updated!
+
+To customize Babel, you have two options:
+
+1. You may create a [`.babelrc`](https://babeljs.io/docs/usage/babelrc/) file in your project's root directory. Any settings you define here will overwrite matching config-keys within [Preact CLI preset]. For example, if you pass a `"plugins"` object, it will replace & reset all Babel plugins that Preact CLI defaults to.
+
+2. If you'd like to modify or add to the existing Babel config, `preact.config.js` must be used (see the [Webpack](#webpack) section for reference) - [here's an example](https://github.com/developit/preact-cli/wiki/Config-Recipes#customising-babel-options-using-loader-helpers) on how to achieve that.
```

Comment:
...existing Babel config, you must use a `preact.config.js` file. Visit the [Webpack](#section) for more info, or check out the [Customize Babel](https://github.com/developit/preact-cli/wiki/Config-Recipes#customising-babel-options-using-loader-helpers) example!